Laurence Qi/Week 2/day1/html-labs/ecardly/css/style.css

```diff
@@ -0,0 +1,98 @@
+/* 
+Project Name: Ecard.ly
+Client: General Assembly 
+Author: Christopher Castig | @castig
+Developer @GA in NYC
+*/
+
+
+/* Box Model Hack */
+
+* {
+    -moz-box-sizing: border-box;
+    /* Firexfox */
+    -webkit-box-sizing: border-box;
+    /* Safari/Chrome/iOS/Android */
+    box-sizing: border-box;
+    /* IE */
+}
+
+
+/* Clear fix hack */
+
+.clearfix:after {
+    content: ".";
+    display: block;
+    clear: both;
+    visibility: hidden;
+    line-height: 0;
+    height: 0;
+}
+
+.clear {
+    clear: both;
+}
+
+#header-background {
+    background-image: url("../images/background.jpg");
+    height: 200px;
+    width: 100%;
+}
+
+body {
+    max-width: 1000px;
+    margin: auto;
+    background-color: rgb(218, 218, 218);
+}
+
+#logo {
+    display: block;
+    margin: auto;
+    margin-top: -175px;
+    padding-top: 20px;
+    padding-bottom: 30px;
+}
+
+
+/* div {
+    background-color: lightgray;
+} */
+
+h2 {
+    margin-top: 0;
+}
+
+#main-block {
+    width: 1000px;
+    margin: auto;
+}
+
```

Comment:
The problem is `float` - this is why you'll see people online telling you to avoid it at costs. Here's what I would do:
```
#main-block {
    width: 1000px;
    margin: auto;
    background-color: white;
    padding: 20px;
    width: 100%;
}

#white {
    display: flex;
    flex-direction: row;
}

#main-img {
    padding-right: 20px;
}
```
See my edit to your index.html as well